src/VkBootstrap.cpp

```diff
@@ -794,6 +794,13 @@ InstanceBuilder& InstanceBuilder::request_validation_layers(bool enable_validati
 	info.request_validation_layers = enable_validation;
 	return *this;
 }
+
+InstanceBuilder & InstanceBuilder::disable_debug_messenger() {
+	info.use_debug_messenger = false;
+	info.debug_callback = nullptr;
```

Comment:
Rather than setting the debug_callback to nullptr, I think it would make more sense to modify the code to not add the debug utils extension if use_debug_messenger is false.

The relevant code change is line 587 of VkBootstrap.c, listed below
```c
	if (info.debug_callback != nullptr && info.use_debug_messenger && system.debug_utils_available) {
		extensions.push_back(VK_EXT_DEBUG_UTILS_EXTENSION_NAME);
	}
```

Reply:
Cause right now `info.use_debug_messenger` is only used to decide if the callback should be setup, not if the extension should be added.

Reply:
I made the change, it works great for my Android